lib/ruby-debug-ide/xml_printer.rb

```diff
@@ -389,9 +392,19 @@ def inspect_with_allocation_control(slice, memory_limit)
       result = nil
       inspect_thread = DebugThread.start {
         start_alloc_size = ObjectSpace.memsize_of_all
```

Comment:
Most often 'inspect' works fast and requires little memory, so in general the performance should not slow down. And in case of heavy inspect-s, when using probability, everything can crush with a high probability. Then why was this fix at all?)